.github/workflows/tests.yaml

```diff
@@ -18,10 +18,10 @@ jobs:
     steps:
     - name: Checkout Code
       uses: actions/checkout@v1
-    - name: Set up JDK 1.8
+    - name: Set up JDK 11
       uses: actions/setup-java@v1
       with:
-        java-version: 1.8
+        java-version: 11
     - uses: actions/cache@v1
```

Comment:
Might want to consider updating checkouts, setup-java, and cache to v3 since this stuff is alredy getting updated.